integration_tests/utils.py

```diff
@@ -13,9 +13,11 @@ def get_repo_root_path():
     return os.path.dirname(rust_vmm_ci_path)
 
 
-def get_cmd_output(cmd):
-    """Returns stdout content of `cmd` command."""
-    cmd_out = subprocess.run(cmd, shell=True, check=True,
-                             stdout=subprocess.PIPE)
-    stdout = cmd_out.stdout.decode('utf-8')
-    return stdout
+def get_cmd_result(cmd, result_type='stdout'):
```

Comment:
I find returning different things based on a function argument a bit confusing. Moreover if you use as the `result_type` value something that is different from `stdout` or `stderr`, then the stdout content will be returned, which is not necessarily right. It would be clearer to return both values, i.e. `return stdout, stderr` at the end.
In this particular case you can also pass [ `check=True` to `subprocess.run()`](https://docs.python.org/3/library/subprocess.html#subprocess.run) and the test will immediately fail (with a decent message that I guess can also be improved) when the command fails.